lib/routes/_update.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const elastic = require('../services/elastic'),
+  helpers = require('../services/elastic-helpers'),
+  responses = require('../services/responses');
+
+/**
+ * Add the prefix to any index specified
+ *
+ * @param  {Object} updateBody
+ * @return {Object}
+ */
+function decorateWithPrefix(updateBody) {
+  if (!updateBody.index) {
+    throw new Error('An index property is required to update');
+  }
+
+  updateBody.index = helpers.indexWithPrefix(updateBody.index);
+
+  return updateBody;
+}
+
+/**
+ * Return a function for `expectJSON` to evaluate which
+ * will update Elastic with the payload from the client's
+ * request
+ *
+ *
+ * @param  {object} updateBody
+ * @return {function}
+ */
+function elasticPassthrough(updateBody) {
+  return function () {
+    var { index, type, id, body } = decorateWithPrefix(updateBody);
```

Comment:
Can you add a check for the `id` and `body` values somewhere? Could do it in `decorateWithPrefix` and then rename function something like `validateUpdateRequest`.